man/src/oxide-bar.1.md

```diff
@@ -0,0 +1,51 @@
+% OXIDE-BAR(1) oxide-bar 0.1.0
+% Philipp Kalinowski
+% February 2023
+
+# NAME
+
+oxide-bar - statusbar for Oxide
+
+# DESCRIPTION
+
+Define the behavior of the statusbar for Oxide. The config file provides the possibility to customize the text and background color of the oxide statusbar.
+As same as the config file for Oxide, the config for the statusbar is written in YAML.
+
+# FILES
+
+When starting, Oxide has to paths it searches for the statusbar config file.
```

Comment:
typo
maybe rephrase to:
During launch, oxide bar searches for a config file in the following locations: